### PR TITLE
Add first scheduled metric

### PR DESCRIPTION
**Pull request by gsvr**
`subscriptions.active.last`

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: seed_stage_based_messaging/settings.py [hunk 1]
```diff
@@ -168,6 +168,9 @@
     'subscriptions.tasks.fire_metrics': {
         'queue': 'metrics',
     },
+    'subscriptions.tasks.scheduled_metrics': {
+        'queue': 'scheduledmetrics',
+    },
```
Comment (imsickofmaps) [inline seed_stage_based_messaging/settings.py]: Please add two lists to settings: `metrics_realtime` and `metrics_scheduled`

File: subscriptions/tasks.py [hunk 2]
```diff
@@ -294,3 +295,33 @@ def run(self, subscription_id, **kwargs):
...
+
+class ScheduledMetrics(Task):
+
+    """ Fires off tasks for all the metrics that should run
+        on a schedule
+    """
+    name = "seed_staged_based_messaging.subscriptions.tasks.scheduled_metrics"
+
+    def run(self):
+        fire_active_last.apply_async()
```
Comment (imsickofmaps) [inline subscriptions/tasks.py]: Iterate through the setting value here

**Comment by imsickofmaps** [comment]
👍

**Comment by imsickofmaps** [comment]
(please add don't store in DB flags)

**Pull request merged**